routes/api_v1.js

```diff
@@ -1,5 +1,5 @@
 const router = require("express").Router();
 const userRoutes =  require("./userRoutes");
 
-router.use("/users", userRoutes);
+router.use("/user", userRoutes);
```

Comment:
let's stick with plural `/users` instead of singular `/user`. 

Search for `REST API URL naming conventions` to see why. Ideally the `get` and `update` and `delete` for single user would be /user and the rest /users but its a common convention to simply use plural for everything. 


For example: `GET .../api/v1/users/1`

Reply:
Ok, I only want to leave it clear that I used singular because that's the format that you had requested in this [issue](https://github.com/ProSchedule-Manager/user-permissions-app/issues/6)

Reply:
So, are you going to leave it as `/user`? If not, please make and push the changes prior to comment in here.

Reply:
No, I have already pushed the changes prior